### ISSUE TITLE
Fatal Error when using display suite

### ISSUE THREAD

**Issue by leonk** [acquia/lightning#254]
Steps to reproduce

1. Install Acquia Lightning (just core, no other lightning plugins enabled).
2. Install display suite 8.x-2.6
3. Configure an entity to use display suite for it's layout.
4. Hit save

You get the following error....
```
Fatal error: Call to a member function getThirdPartySetting() on null in /var/www/uniper/docroot/profiles/contrib/lightning/modules/lightning_features/lightning_core/lightning_core.module on line 42
```

**Comment by leonk** [comment]
Just to confirm, this is happening with the latest version of Lightning (8.1.15)

**Comment by balsama** [comment]
Confirmed that this bug exists with HEAD of 2.x Lightning and 2.x DS.

(@leonk - incidentally, 2.0.1 is latest version of Lightning. It's functionally equal to 1.15, but uses the Drupal.org packagist. The 1.x branch won't have any more releases)

**Comment by phenaproxima** [comment]
I am reproducing this as well and will have a fix soon.

**Comment by phenaproxima** [comment]
This is now fixed in the 8.x-ds-bug branch.

**Comment by phenaproxima** [comment]
Fix merged in 23d521bda543dba92dd49260117f591e0ab82985.
